### PR TITLE
Fix typo in mbedtls_ssl_set_bio description.

### PR DESCRIPTION
**Pull request by cm6n**
## Description
Description referred to mbedtls_ssl_sent_t callback,
but the callback is named mbedtls_ssl_send_t.

Signed-off-by: Christopher Moynihan <christophm@gmail.com>

## Todos
- [x] Changelog updated
- [x] Backports: https://github.com/ARMmbed/mbedtls/pull/3660 https://github.com/ARMmbed/mbedtls/pull/3661

### CODE REVIEW

File: ChangeLog.d/comment_typo_in_mbedtls_ssl_set_bio.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Changes
+   * Fix comment typo in mbedtls_ssl_set_bio_description.
```
Comment (hanno-arm) [inline ChangeLog.d/comment_typo_in_mbedtls_ssl_set_bio.txt:2]: Minor: Remove the underscore. 
```suggestion
   * Fix comment typo in documentation of mbedtls_ssl_set_bio.
```

**Comment by cm6n** [comment]
Good point! It's the the Law of Conservation of Typos.

**Review by hanno-arm — APPROVED** [review]
LGTM

**Review by bensze01 — APPROVED** [review]

**Comment by bensze01** [comment]
Considering the PR is only correcting a typo in the documentation, the CI failures look unrelated.

**Comment by bensze01** [comment]
The `continuous-integration/jenkins/pr-merge` job also completed, even though it seems stuck on "in progress" on Github.

**Comment by bensze01** [comment]
Actually, this typo is present in both the mbedlts-2.7 and mbedtls-2.16 LTS branches. If you could backport it, @cm6n , that whould be great (A cherry-pick should apply cleanly).

**Comment by bensze01** [comment]
Since they were trivial, I did the backports myself: #3660, #3661 .

**Comment by cm6n** [comment]
Thank you Bence!

>

**Comment by bensze01** [comment]
All Jenkins tests except the Mbed OS ones passed. The Travis ones only failed because it timed out, but it is a subset of Jenkins, so this should be ready to merge.

**Comment by gilles-peskine-arm** [comment]
pr-merge had glitched. A rerun of pr-merge passed, but didn't report to GitHub properly.

Travis had glitched, so I reran it yesterday. I triggered a rebuild from the Travis interface. Apparently, this reused the same `.travis.yml` as the previous run on that PR, but it checked out a refreshed merge (the target branch had changed in the meantime). One of the changes on `development` was the Travis job “check compilation guards” started taking much longer (due to testing more configurations), too long for a Travis job. So we removed that job from Travis (it's still running on Jenkins). But because this was a rebuild and not a new GitHub trigger event, Travis didn't take parse the new `.travis.yml`, and therefore it ran the “check compilation guards”. This is an edge case due to using the “rebuild” button on Travis, not a problem we can expect to run into on a regular basis.

**Pull request merged**